web_src/css/modules/modal.css

```diff
@@ -47,7 +48,8 @@ These inconsistent layouts should be refactored to simple ones.
 }
 
 .ui.modal > .actions,
-.ui.modal .content + .actions {
+.ui.modal .content + .actions,
+.ui.modal .content + form > .actions {
```

Comment:
Are these fomantic's selectors or our own?

Reply:
I see, well it could be simplified to `.content .actions`, but leaving that up to you.

Reply:
I do not feel it is safe to use `content` or `actions` without a strict rule. It would cause pollution because many one-word classes (including "content") have been used everywhere.

Reply:
Another (strong) reason is that the `content` itself has paddings, so the `actions` inside it shouldn't have padding again. But the `actions` following it should still have the padding.

Declaring these style explicitly is good for maintainability.